### DISCUSSION TITLE
OpenCV not working on CODE app

### DISCUSSION THREAD

**Discussion by Manjunath-Aero** [thebaselab/codeapp#978] [Q&A]
Hello,
I want to run some opencv programs on my ipad, but when I try to run the code, on terminal it is showing that “modulenotfounderror: no module named cv2”. I tried to install opencv by typing in “pip install opencv-python” but then also I am getting a series of error messeges as shown in the attached screenshot.
![IMG_1564](https://github.com/thebaselab/codeapp/assets/71602680/88b26c5d-9431-448c-8628-9a79b36d4e33)
![IMG_1565](https://github.com/thebaselab/codeapp/assets/71602680/3390e6ea-67d4-4dcb-8f39-7156bca33a01)

After some research, I got to know that opencv can’t be installed if the python version is more than 3.7 and now I want to downgrade the installed python version, how can I do that?

Any quick help from the community will be appreciated. Please help me!

Thank you

**Comment by bummoblizard** [comment]
OpenCV is not supported in Code App's local environment due to technical constraints. You may consider connecting to an external ssh server to get access to a full desktop environment. To read more: https://code.thebaselab.com/guides/connecting-to-a-remote-server-ssh-ftp
